projects/fsastorefrontstyles/scss/overrides/homepage.scss

```diff
@@ -2,39 +2,6 @@
 @import '../theme';
 
 .LandingPageTemplate {
-  .enriched-banner {
```

Comment:
Shouldn't we remove the style from `FSCategoryPageTemplate` as well?

Reply:
Yes, but enriched-banner from `LandingPageTemplate` and `FSCategoryPageTemplate` don't have the same style, I can remove only few styles from `FSCategoryPageTemplate`.

Reply:
Yes, but that then defeats the purpose of this task. We should have one general and centralized style for every enriched banner. The rest should be defined via this feature.